pkg/prompts/prompts.go

```diff
@@ -244,3 +272,40 @@ func Select[T any](label string, items []T, opt *SelectOpt[T]) (T, error) {
 
 	return items[i], nil
 }
+
+func getCurrentDirName() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("getting current directory: %v", err)
+	}
+	return filepath.Base(dir), nil
+}
+
+// Sanitize the directory name to comply with k8s label rules
+func sanitizeAppName(name string) string {
+	// Remove all characters except alphanumeric, '-', '_', '.'
+	reg := regexp.MustCompile(`[^a-zA-Z0-9-_.]+`)
+	sanitized := reg.ReplaceAllString(name, "")
+
+	// Trim leading and trailing '-', '_', '.'
+	sanitized = strings.Trim(sanitized, "-._")
+
+	// If name is empty, return empty string
+	if sanitized == "" {
+		return ""
+	}
+
+	// Ensure appname starts with alphanumeric characters
+	regStart := regexp.MustCompile(`^[^a-zA-Z0-9]+`)
+	sanitized = regStart.ReplaceAllString(sanitized, "")
+
+	// Ensure appname ends with alphanumeric characters
+	regEnd := regexp.MustCompile(`[^a-zA-Z0-9]+$`)
```

Comment:
same here, you can potentially use [IsLetter](https://pkg.go.dev/unicode#IsLetter) here for clarity- then we don't need comments for each regex explaining what they do